### PR TITLE
use correct encoding

### PR DESCRIPTION
**Pull request by bubenkoff**
[Review on Reviewable](https://reviewable.io/reviews/gabrielfalcao/httpretty/225)

### CODE REVIEW

**Comment by bubenkoff** [comment]
Please consider merging and making a new release - ATM it doesn't install

File: httpretty/__init__.py [hunk 1]
```diff
@@ -25,32 +25,35 @@
...
+    PATCH = httpretty.PATCH
+    OPTIONS = httpretty.OPTIONS
+    CONNECT = httpretty.CONNECT
+    
+    
+    def last_request():
+        """returns the last request"""
+        return httpretty.last_request
+except ImportError:
+    pass
```
Comment (gabrielfalcao) [inline httpretty/__init__.py]: I'm not sure if this `try/except` is a good thing. That will cause a `NameError` if the imports fail. A misleading exception, I would say

**Comment by bubenkoff** [comment]
now it should be fine

File: setup.py [hunk 2]
```diff
@@ -24,29 +24,16 @@
...
-
-
 def read_version():
     """Read version from httpretty/version.py without loading any files"""
-    finder = VersionFinder()
-    finder.visit(
-        ast.parse(local_file('httpretty', '__init__.py').encode('utf-8')))
-    return finder.version
+    contents = local_file('httpretty', '__init__.py')
+    return re.compile(r".*__version__ = version = '(.*?)'", re.S).match(contents).group(1)
```
Comment (gabrielfalcao) [inline setup.py]: this is a lot less reliable than using the python's abstract syntax tree, can we keep it the old way ? :)

**Comment by gabrielfalcao** [comment]
obsolete

**Pull request closed without merging**